### PR TITLE
MINOR, WIP remove warning for custom solvers in rERP

### PR DESCRIPTION
**Pull request by jona-sassenhagen**
Since .15, linear_regression_raw warns if a custom solver is passed. This can probably be removed for .17.

### CODE REVIEW

File: mne/stats/regression.py [hunk 1]
```diff
@@ -253,11 +253,7 @@ def solver(X, y):
...
                                     overwrite_a=True, overwrite_b=True).T
     elif callable(solver):
-        warn("When using a custom solver, note that since MNE 0.15, this "
-             "function will pass the transposed data (n_channels, n_times) "
-             "to the solver. If you are using a solver that expects a "
-             "different format, it will give wrong results and might in "
-             "extreme cases crash your session.")
+        pass
     else:
         raise TypeError("The solver must be a str or a callable.")
```
Comment (larsoner) [inline mne/stats/regression.py]: Simpler now as `elif not callable(solver):` but we can let it slide :)

**Comment by agramfort** [comment]
needs rebase to become green?

**Comment by jona-sassenhagen** [comment]
Given
>Your current usage represents 646% of your mne-tools Linux plan's limit. Please upgrade in order to ensure no disruption in building.

@larsoner should I rebase and run the Circle test again ..?

I think it doesn't actually touch any examples, so I don't know if it will count against the limit.

**Comment by larsoner** [comment]
No need. Feel free to merge if you are done @jona-sassenhagen

**Comment by jona-sassenhagen** [comment]
Thanks guys

**Comment by larsoner** [comment]
>  I don't know if it will count against the limit.

Even when no examples are changed, it builds the `make html_noplot` which takes ~20 minutes including time to set up the venv.

**Pull request merged**